docs/src/assets/images/trustedTeamsLogos/index.ts

```diff
@@ -0,0 +1,93 @@
+import LodestarImage from './lodestar.png';
```

Comment:
let's move this file from assets to somewhere closer to component

Reply:
e.g. it could be `docs/src/components/landingBlocks/TrustedTeamsBlock.model.tsx`

Reply:
Done